File: problems/0004_median-of-two-sorted-arrays/concat-sort/Solution.swift
class Solution {
    func findMedianSortedArrays(_ nums1: [Int], _ nums2: [Int]) -> Double {
        var nums = nums1 + nums2
        nums.sort()
        let i = nums.count / 2
        return (nums.count % 2 == 0 ? Double(nums[i - 1] + nums[i]) / 2.0 : Double(nums[i]))
    }
}

File: problems/0004_median-of-two-sorted-arrays/concat-sort/Solution.php
<?php
class Solution {
    /**
     * @param Integer[] $nums1
     * @param Integer[] $nums2
     * @return Float
     */
    function findMedianSortedArrays($nums1, $nums2) {
        $nums = [];
        array_push($nums, ...$nums1);
        array_push($nums, ...$nums2);
        sort($nums);
        $size = count($nums);
        $i = intdiv($size, 2);
        return ($size % 2 == 0 ? ($nums[$i - 1] + $nums[$i]) / 2 : $nums[$i]);
    }
}
